etn-ccis/blui-react-workflows
# Added success screen default in forgot password base screen

Fixes # BLUI-4383



#### Changes proposed in this Pull Request:

- Added success screen default value in forgot password base screen



#### Screenshots / Screen Recording (if applicable)
<img width="1792" alt="Screenshot 2023-07-05 at 3 55 31 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/120575281/4c9b2845-744c-48c6-b191-2aa8b91226bd">



#### To Test:

- cd login-workflow
- yarn && yarn build
- yarn link:workflow2
- yarn start:example2
- Click on DEBUG button
- Click on Forgot Password Full Screen button
- Enter any valid email in the field and click on next
- You should be able to see the success screen



#### Any specific feedback you are looking for?

-
